content/community/organization/policies.md

```diff
@@ -0,0 +1,110 @@
+
+### Formal Releases
+
+The Haiku project aims to make an annual release of the operating system.  In general a formal release will tend to be made in the northern hemisphere spring / southern hemisphere autumn.  The rationale for this timing is because it is the time of year at which the development team are most likely to have capacity to deal with a release.  At the release time, the development team will attempt to stabilize the state of the project and to create a reference point for users of the operating system.  A release may be deferred or may be cancelled entirely if there are difficulties making the release for any reason.
+
+### Coding Guidelines
+
+The information in the Haiku [Coding Guidelines](/development/coding-guidelines) is most important to the project.
+
+Some code in the project does not match the guidelines. The non-compliant code is largely due to such material being authored before the guidelines were published. Assistance with modifying the non-compliant material to reach compliance is welcome as long as the related commit documents that the changes are related to guideline-compliance and do not contain functional changes.  Take care not to mix functional changes into a commit that is otherwise has the purpose of improving guideline compliance.
+
+If you will be contributing code or patches to Haiku, you will need to strictly follow the coding guidelines. Code which does not follow these guidelines will not be accepted. The guidelines have been in place for a long time affecting the whole code base and there is little appetite for discussing changes to the coding guidelines.
+
+### User Demands
+
+Some community members are occasionally frustrated that a feature or function that they would like is not available on the platform or is not sufficiently developed for their actual or perceived purpose.  In such situations, keep in mind that the contributors to the project are doing so mostly on a volunteer basis and because the work interests them.  Although often times, contributors are interested to hear about any problems that users are having, the contributors are not beholden to users' requirements.
+
+If you are unable to contribute materially to the software yourself, the best way that you can achieve the goal of having a feature implemented is by using the [bug tracker](https://dev.haiku-os.org) system.  Identify if there is an existing ticket that correlates with your requirements or if there is no existing ticket then create one.  You should document factually, clearly and accurately the problem you have.  Make sure there is enough detail that the problem can easily be understood and reproduced by a contributor.  If there are obvious resources that might help such as input / output files, screenshots, logs etc... then attach those to the ticket directly.
```

Comment:
The bugtracker already has documentation on how to write a good bug report and there is no need to repeat it here. IT kind of loses the main point, which is that support is provided by volunteers in a best-effort way only, and it can be frustrating because it causes large delays in handling of tickets.

Other things could be clarified however:
- The possibility to vote on existing tickets to raise the attention of the devs,
- The bountysource page if people want to put money on a specific ticket,
- The fact that donations to Haiku inc are only donations and do not mean we owe them anything. In particular, the Inc and the Haiku project are independant entities and there are alternatives (the HSA, direct donations to developers on Liberapay, ...)